pkgs/development/python-modules/pywick/default.nix

```diff
@@ -28,18 +31,13 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [
-    h5py hickle numpy pandas pillow six torch torchvision tqdm
+    h5py hickle numpy opencv4 pandas pillow prodict torch torchvision tqdm six yacs
   ];
 
-  checkInputs = [ pytest ];
-
-  checkPhase = ''
-    runHook preCheck
-    pytest tests/
-    runHook postCheck
-  '';
+  checkInputs = [ pytestCheckHook ];
 
   meta = {
+    broken = true;  # requires albumentations (stalled PR: https://github.com/NixOS/nixpkgs/pull/165715)
```

Comment:
You can just cherry-pick the commit if you like. Also meta.broken should be last.

Reply:
I've moved meta.broken to last.

I tried to cherry-pick from the other PR but it looks like there's also a transitive dependency on `python3Packages.imgaug` which is broken right now until it adds `scikitimage` 0.19 compatibility.